session.go

```diff
@@ -2369,6 +2369,11 @@ func (c *Collection) NewIter(session *Session, firstBatch []bson.Raw, cursorId i
 		timeout: -1,
 		err:     err,
 	}
+
+	if socket.ServerInfo().MaxWireVersion >= 4 && c.FullName != "admin.$cmd" {
```

Comment:
It's a good question. This is the inverse check of what's on line 3225: https://github.com/craiggwilson/mgo/blob/c283eb40dfc206fd4c5f608663f10d14e646286d/session.go#L3225. I thought it best to use the same condition before setting this variable.